analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -519,15 +519,15 @@ open class Npm(
             return null
         }
 
-        log.debug { "Building dependency tree for '${moduleInfo.name}' from directory '$moduleDir'." }
+        log.trace { "Building dependency tree for '${moduleInfo.name}' from directory '$moduleDir'." }
```

Comment:
We don't actually use the "trace" level in ORT yet (besides a single use in the contributed `OpossumReporter`; I'm going to remove that single use), and we don't even allow user to set the "trace" level in `OrtMain`. So How about simply dropping these?

Reply:
> I'm going to remove that single use

See https://github.com/oss-review-toolkit/ort/pull/4825.